README.md

```diff
@@ -187,4 +187,7 @@ request dated after any votes resets those votes.
 If the rule-change is adopted, a player who can shall merge the pull-request
 in a timely fashion, which marks completion of the vote.
 
-
+**301** If a PR that outlines a rule-change (in accordance with other rules in
+this file) contains `y` commit hashes containing an English dictionary word of 
+at least four consecutive letters, the proposer will be awarded `y` additional 
+points upon adoption of the rule-change.
```

Comment:
Agree. This makes more sense with respect to the dependency between PR and rule-change.